readthedocs/oauth/services/github.py

```diff
@@ -100,11 +100,22 @@ def create_repository(self, fields, privacy=None):
             (fields['private'] is False and privacy == 'public'),
         ]):
 
-            repo, _ = RemoteRepository.objects.get_or_create(
+            repo, created = RemoteRepository.objects.get_or_create(
                 remote_id=str(fields["id"]),
                 vcs_provider=self.vcs_provider_slug,
             )
 
+            if created:
+                _old_remote_repository = RemoteRepository.objects.filter(
+                    full_name=fields["full_name"], vcs_provider=self.vcs_provider_slug
```

Comment:
Shouldn't you omit the current remote id? But I guess if this was just created, then it doesn't have a full_name yet?

Reply:
I'm tempted to merge this as is to get it out in today's release. Tests aren't breaking, but this line might not be tested 🙃

Reply:
```suggestion
                    full_name=fields.get("full_name"), vcs_provider=self.vcs_provider_slug
```

Reply:
re-read the comment, and I think you're saying this code should work? Going head & merging.

Reply:
`fields` is the dictionary response from the request. So, all the attribute exist there.

The idea is to find out "in what cases we are creating a new `RemoteRepository` with different `remote_id` but equal `full_name`".